src/model/objection.session.extra.data.ts

```diff
@@ -0,0 +1,6 @@
+import ObjectionCompanyProfile from "./objection.company.profile";
+
+export default interface ObjectionSessionExtraData {
+  objections_company_profile?: ObjectionCompanyProfile;
```

Comment:
Just 'objection_company_profile'? We should really tidy all these artefact names up and decide on a standard.